cms/server/admin/handlers/base.py

```diff
@@ -68,7 +69,14 @@
 
 logger = logging.getLogger(__name__)
 
-
+"""
+#TODO improve and implement this function
+def natural_sort_key(text):
+    #Generate a natural sorting key for alphanumeric strings.
+    def convert(part):
+        return int(part) if part.isdigit() else part.lower()
+    return [convert(c) for c in re.split("([0-9]+)", str(text))]
+"""
```

Comment:
I think the TODO is enough. Let's remove this commented code, we can add it later (and it's easy to retrieve by tracing back to this PR following the git history).